### ISSUE TITLE
Empty tree in CLI with version intervals

### ISSUE THREAD

**Issue by alexarchambault** [coursier/coursier#1356]
With `2.0.0-RC3-3`:
```bash
$ cs resolve -t ammonite:latest.stable
  Result:
└─ com.lihaoyi:ammonite_2.13.0:latest.stable -> 1.6.9 (possible incompatibility)

$ cs resolve -t 'ammonite:1.6+'
  Result:
└─ com.lihaoyi:ammonite_2.13.0:1.6+ -> 1.6.9-18-1b78002b (possible incompatibility)

$ cs resolve ammonite:1.6.9
└─ com.lihaoyi:ammonite_2.13.0:1.6.9
   ├─ com.lihaoyi:ammonite-interpApi_2.13.0:1.6.9
   │  ├─ com.lihaoyi:ammonite-ops_2.13:1.6.9
   │  │  ├─ com.lihaoyi:os-lib_2.13:0.3.0
   │  │  │  └─ com.lihaoyi:geny_2.13:0.1.8
   │  │  └─ org.scala-lang.modules:scala-collection-compat_2.13:2.0.0
…
```

**Comment by dwijnand** [comment]
Looks fixed, using coursier 2.0.0-RC6-8

```
$ cs resolve -t ammonite:latest.stable
  Result:
└─ com.lihaoyi:ammonite_2.13.1:latest.stable -> 2.0.4 (possible incompatibility)
   ├─ com.lihaoyi:ammonite-interp-api_2.13.1:2.0.4
   │  ├─ com.lihaoyi:ammonite-ops_2.13:2.0.4
...
```

**Comment by alexarchambault** [comment]
Indeed, that's fixed. (Don't remember what fixed that…)
